### DISCUSSION TITLE
Has anyone successfully installed Nextcloud AIO on RHEL (or one of its derivatives)?

### DISCUSSION THREAD

**Discussion by Verhoeckx** [nextcloud/all-in-one#1735] [Questions] [answered]
Hello Nextcloud AIO users and developers,

Question:
Has anyone successfully installed Nextcloud AIO on RHEL (or one of its derivatives)?

Why? Well I'm planning to install Nextcloud AIO on RHEL and was wondering which problems I could expect?

And if so:
- Did you use Docker or Podman?
If you ran Docker, how did you install it? Did run into any issues with Podman?
- Did you have any firewall problems (besides the one mention in the documentation)?
- Did you have SELinux issues?

Thanks in advance!
I'm planning to document all the steps that I'm taking (in order to help others).

Yours sincerely,

Verhoeckx

**Comment by szaimen** [comment, marked as answer]
> Has anyone successfully installed Nextcloud AIO on RHEL (or one of its derivatives)?

I myseelf did not try yet but I think many are already running it successfully on RHEL or one of its derivates.

> * Did you use Docker or Podman?

Only Docker is supported as you know already.

>   If you ran Docker, how did you install it? Did run into any issues with Podman?

I suppose following the official docker documentation should be sufficient.

> * Did you have SELinux issues?

Disabling SELinux in the beginning and enabling it after verifying that everything works should be the best way to allow to debug potential problems with SELinux.

**Comment by Verhoeckx** [comment]
> Only Docker is supported as you know already.

I read that if you install Docker and Podman simultaneously on RHEL (or one of its derivatives) you can run in all kind of troubles. Because Nextcloud AIO only supports Docker, someone on RHEL is now obliged to use Docker for all the other containers. If Nextcloud AIO would support Podman, someone on RHEL keeps the freedom to use Docker or Podman. Having said that there is a [workaround](https://faun.pub/how-to-install-simultaneously-docker-and-podman-on-rhel-8-centos-8-cb67412f321e) and it might be [fixed](https://github.com/docker/containerd-packaging/pull/231) in the future. And there are others [that claim](https://github.com/docker/containerd-packaging/issues/210) that it does work.

>Disabling SELinux in the beginning and enabling it after verifying that everything works should be the best way to allow to debug potential problems with SELinux.

Thanks, that's a good idea!
